src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyDeviceChallengePoll.ts

```diff
@@ -223,7 +223,6 @@ export const transformOktaVerifyDeviceChallengePoll: IdxStepTransformer = ({
 
   if ([
     CHALLENGE_METHOD.APP_LINK,
-    CHALLENGE_METHOD.CHROME_DTC,
```

Comment:
Why this one is removed?

Reply:
For `CHROME_DTC` challenge method the `titleElement` is of type `ActionPending`
https://github.com/okta/okta-signin-widget/blob/e0fe1307ecb66ded51c1b59164a2a290faa84ebd/src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyDeviceChallengePoll.ts#L43-L50
https://github.com/okta/okta-signin-widget/blob/e0fe1307ecb66ded51c1b59164a2a290faa84ebd/src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyDeviceChallengePoll.ts#L168

`ActionPending` already includes `Spinner ` so we should not add another one at line 228
https://github.com/okta/okta-signin-widget/blob/e0fe1307ecb66ded51c1b59164a2a290faa84ebd/src/v3/src/transformer/layout/oktaVerify/transformOktaVerifyDeviceChallengePoll.ts#L228